docroot/themes/humsci/su_humsci_theme/templates/views/views-view-unformatted.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Theme override to display a view of unformatted rows.
+ *
+ * Available variables:
+ * - title: The title of this group of rows. May be empty.
+ * - rows: A list of the view's row items.
+ *   - attributes: The row's HTML attributes.
+ *   - content: The row's content.
+ * - view: The view object.
+ * - default_row_class: A flag indicating whether default classes should be
+ *   used on rows.
+ *
+ * @see template_preprocess_views_view_unformatted()
+ */
+#}
+{% if title %}
```

Comment:
Is this a safe condition to rely on? Could there be other reasons that the title is present, or, could there be a time when a group doesn't have a title?

Reply:
This came from `stable` theme in core. i assume its safe.

Reply:
https://git.drupalcode.org/project/drupal/blob/8.8.x/core/themes/stable/templates/views/views-view-unformatted.html.twig#L18

Reply:
Ah, you're only adding a div. 
More divitis for Drupal views but meh.